src/main/java/uk/gov/digital/ho/hocs/document/DocumentDataResource.java

```diff
@@ -105,10 +104,10 @@ private static ResponseEntity<ByteArrayResource> generateFileResponseEntity(S3Do
                 .body(resource);
     }
 
+    @Deprecated(forRemoval = true)
```

Comment:
Usages have already been changed? Might as well remove and major bump it? Unless you want to wait a week or so?

Reply:
I've got a separate PR for the removal, maybe merge it in with the EntityActionUUID change?